scripts/zones/Windurst_Woods/npcs/Mheca_Khetashipah.lua

```diff
@@ -4,13 +4,21 @@
 -- Type: Standard NPC
 -- !pos 66.881 -6.249 185.752 241
 -----------------------------------
+require("scripts/globals/quests")
+-----------------------------------
 local entity = {}
 
 entity.onTrade = function(player, npc, trade)
 end
 
 entity.onTrigger = function(player, npc)
-    player:startEvent(426)
+    starStatus = player:getQuestStatus(tpz.quest.log_id.WINDURST, tpz.quest.id.windurst.MIHGO_S_AMIGO)
```

Comment:
local

Reply:
Bah!  And here I thought I reviewed through everything.  Corrected.